fdbcli/CoordinatorsCommand.actor.cpp

```diff
@@ -100,6 +100,7 @@ ACTOR Future<bool> changeCoordinators(Reference<IDatabase> db, std::vector<Strin
 				state std::vector<StringRef>::iterator t;
 				for (t = tokens.begin() + 1; t != tokens.end(); ++t) {
 					try {
+						// TODO(renxuan): add hostname parsing here.
```

Comment:
I have a bold ask here: at least turn the newly added `TODO`s into actual GitHub issues and properly screen them on a regular bases.

Let's admit this: we often add `TODO`s and forget about them. I just grep through the main branch and found 475 `TODO`s in the code base.

Reply:
Very good point, thank you for the reminder! This is actually what I AM working on, it's just for bringing reviewers' attention that we will have a code change here, to give everyone a better whole picture, just like I did [here](https://github.com/apple/foundationdb/blob/334ba66348cd3b347148d507bf90d05a161fe5f1/fdbserver/SimulatedCluster.actor.cpp#L1908-L1909) yesterday, I eliminate the TODO today in this PR.